windows-lb/compute.tf

```diff
@@ -1,12 +1,22 @@
+## Copyright © 2021, Oracle and/or its affiliates. 
+## All rights reserved. The Universal Permissive License (UPL), Version 1.0 as shown at http://oss.oracle.com/licenses/upl
 
 /* Instances */
 
 resource "oci_core_instance" "instance1" {
-  availability_domain = random_shuffle.compute_ad.result[1 % length(random_shuffle.compute_ad.result)]
+  availability_domain = var.availablity_domain_name == "" ? data.oci_identity_availability_domains.ADs.availability_domains[0]["name"] : var.availablity_domain_name
```

Comment:
These instances can not be in the same availability domain. That’s why these was a random shuffle. So single AD regions will shuffle in to the same AD, and multi AD regions will use multiple ADs

Reply:
@jeevanjoseph The idea was to give customers a choice of AD. If you are against I will return to random_shuffle.

Reply:
@jeevanjoseph after a call I will revert it back to random_shuffle.